trade-finance-manager-api/src/v1/controllers/deal.tasks.api-test.js

```diff
@@ -108,6 +137,9 @@ describe('createDealTasks', () => {
 
     describe('when no additional tasks should be added', () => {
       it('should return empty array', () => {
+        mockDealWithPartyUrn.tfm.parties.buyer = null;
+        mockDealWithPartyUrn.tfm.parties.exporter = { partyUrn: 'Test ' };
+        mockDealWithPartyUrn.dealType = CONSTANTS.DEALS.DEAL_TYPE.GEF;
         const result = listAdditionalTasks(mockDealWithPartyUrn);
```

Comment:
Can an additional test be added with empty string but with empty spaces i.e. '  ' to test the trim functionality please.